wk5/5.a/memorygame.js

```diff
@@ -45,19 +45,20 @@ function setup() {
     }
 }
 
-function draw ()
-    background(0); {
+function draw() {
+    console.log("(re)sumed draw")
+    background(0); 
     if (gameState.numMatched === gameState.totalPairs) {
         fill('yellow');
-        textsize(66);
+        textSize(66);
         text('you win!!!!!', 400, 425);
         noLoop();
     }
     for (let k = 0; k < cards.length; k++) {
-        if(!card[k].isMatch) {
-            cards[k].face = DOWN;
+        if(!cards[k].isMatch) {
```

Comment:
this is fiddly too! cards vs card! cards is the name of the array so this needed to be `cards[k]`